### ISSUE TITLE
support Remote RSS for QBittorent

### ISSUE THREAD

**Issue by lozbrown** [erickok/transdroid#482]
Currently uTorrent and possibly others are supported for remote RSS

QBittorent webAPI now supports RSS
https://github.com/qbittorrent/qBittorrent/wiki/WebUI-API-(qBittorrent-4.1)#rss-experimental

Can this be included for support in the QBittorent adapter

**Comment by gggirlgeek** [comment]
Yes, please!

**Comment by twig** [comment]
FYI that API link doesn't work

**Comment by erickok** [comment]
Yes it's https://github.com/qbittorrent/qBittorrent/wiki/WebUI-API-(qBittorrent-4.1)#rss-experimental
